src/main/java/com/gluonhq/substrate/SubstrateDispatcher.java

```diff
@@ -116,9 +118,13 @@ public static void main(String[] args) throws Exception {
         }
         System.err.println("Running...");
         if (expected != null) {
-            InputStream is = targetConfiguration.run(paths.getAppPath(), appName);
-            // TODO: compare expected and actual output
-
+            String response = targetConfiguration.run(paths.getAppPath(), appName);
```

Comment:
What if the output contains an (almost) infinite number of lines? In case the result is an inputstream, there would be no issues as long as the input is not read, but in case of a String, it will use memory.

Reply:
We use a buffer to read the input stream. When the process ends, we return the last line read.
Do you suggest we put a limit in the buffer size and only store a given number of lines?